### PR TITLE
Let user run PHP-CS-Fixer 2 and fix CS

### PR DESCRIPTION
**Pull request by lucasmichot**
Let user run PHP-CS-Fixer 2 and fix CS

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "require-dev": {
         "php-vcr/phpunit-testlistener-vcr": "^1.1",
         "jakub-onderka/php-parallel-lint": "^0.9.2",
-        "jakub-onderka/php-console-highlighter": "^0.3.2"
+        "jakub-onderka/php-console-highlighter": "^0.3.2",
+        "friendsofphp/php-cs-fixer": "~2"
```
Comment (realityking) [inline composer.json]: Same as with phpunit, let's not add dependencies just for convenience's sake.

Comment (lucasmichot) [inline composer.json]: Sure, nevertheless this allows people to provide consistent CS on PR they submit.
Also projects require this library use the `--no-dev` during the build process, so this dependencies will be excluded.

In this case in might be complicate for a developer to install the right version of php-cs-fixer (~1 and ~2) are not compatibles, and not force them to install a specific version globally

Comment (realityking) [inline composer.json]: I agree that the 1 vs 2 will be an issue here but the rule of "anything that can be a phar should be" is not one I'm willing to compromise on. Many of the same dependencies used by these tools are also used by code generation libraries and one of those will probably end up in this SDK at some point. With PHP/composer only supporting a single version for each library I'd rather be extra careful and avoid version clashes.

Comment (lucasmichot) [inline composer.json]: Sure

**Pull request closed without merging**